docs/ibc/overview.md

```diff
@@ -6,170 +6,186 @@ parent:
 
 For this reason, most modules should use UNORDERED channels as they require less liveness guarantees to function effectively for users of that channel.
 
-### [Acknowledgements](https://github.com/cosmos/ibc-go/blob/main/modules/core/04-channel)
+### [Acknowledgments](https://github.com/cosmos/ibc-go/blob/main/modules/core/04-channel)
 
-Modules may also choose to write application-specific acknowledgements upon processing a packet. This may either be done synchronously on `OnRecvPacket`, if the module processes packets as soon as they are received from IBC module. Or they may be done asynchronously if module processes packets at some later point after receiving the packet.
+Modules can also choose to write application-specific acknowledgments upon processing a packet. This acknowledgment can be done synchronously on `OnRecvPacket` if the module processes packets as soon as they are received from IBC module. Or acknowledgments can be done asynchronously if module processes packets at some later point after receiving the packet.
 
-Regardless, this acknowledgement data is opaque to IBC much like the packet `Data` and will be treated by IBC as a simple byte string `[]byte`. It is incumbent on receiver modules to encode their acknowledgemnet in such a way that the sender module can decode it correctly. This should be decided through version negotiation during the channel handshake.
+Regardless, this acknowledgment data is opaque to IBC much like the packet `Data` and is treated by IBC as a simple byte string `[]byte`. It is incumbent on receiver modules to encode their acknowledgment so that the sender module can decode it correctly. This should be decided through version negotiation during the channel handshake.
 
-The acknowledgement may encode whether the packet processing succeeded or failed, along with additional information that will allow the sender module to take appropriate action.
+The acknowledgment can encode whether the packet processing succeeded or failed, along with additional information that allows the sender module to take appropriate action.
 
-Once the acknowledgement has been written by the receiving chain, a relayer will relay the acknowledgement back to the original sender module which will then execute application-specific acknowledgment logic using the contents of the acknowledgement. This may involve rolling back packet-send changes in the case of a failed acknowledgement (refunding senders).
+After the acknowledgment has been written by the receiving chain, a relayer relays the acknowledgment back to the original sender module which  then executes application-specific acknowledgment logic using the contents of the acknowledgment. This can involve rolling back packet-send changes in the case of a failed acknowledgment (refunding senders).
```

Comment:
The unclear reference (the notorious naked "This") here makes me wonder what the subject is (I couldn't even guess... "This logic"? or "This relay"?)

please r/w or provide the subject of "This" in  

"This can involve rolling"

Reply:
I think it refers to the execution of the acknowledgment logic. So maybe something like `This acknowledgment logic can involve rolling...`.

There is also an extra space in `which  then`.

Reply:
Maybe we should replace that sentence with:

`Application developers can roll back packet-send changes in the case of a failed acknowledgement (e.g. refunding senders in ICS20).`

Reply:
yes better, the more I read this content, the move love it needs!